tests/xharness/Jenkins.cs

```diff
@@ -2321,7 +2297,7 @@ <li>Toggle visibility
 					if (runningTests.Any ()) {
 						writer.WriteLine ($"<h3>{runningTests.Count ()} running tests:</h3>");
 						foreach (var test in runningTests) {
-							writer.WriteLine ($"<a href='#test_{test.TestName}'>{test.TestName} ({test.Mode})</a> {test.Duration.ToString ()} {HtmlFormat ("\n\t" + test.ProgressMessage)}<br />");
+							writer.WriteLine ($"<a href='#test_{test.TestName}'>{test.TestName} ({test.Mode})</a> {test.Duration.ToString ()} {"\n\t" + test.ProgressMessage.AsHtmlFormat ()}<br />");
```

Comment:
Not sure if that really change something in the web view but...

That's not quite the same:
`HtmlFormat ("\n\t" + test.ProgressMessage)`
versus:
`"\n\t" + test.ProgressMessage.AsHtmlFormat ()`

The latter won't format as HTML the `"\n\t"` while the former would.

Reply:
better change it to be sure is the same output.